style.css

```diff
@@ -423,3 +450,188 @@ span {
 }
 
 /* contact form end */
+
+/* Desktop Version Start */
+
+@media only screen and (min-width: 992px) {
+  .desk-nav {
+    display: block;
+    display: flex;
+  }
+
+  .desktop-image {
+    display: block;
+  }
+
+  .lang-item {
+    display: block;
+  }
+
+  .desk-nav li {
+    margin: 0 10px;
+  }
+
+  .mobile-nav {
+    display: none;
+  }
+
+  .text-2 {
+    width: 90%;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    margin: 48px 0;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .introduction {
+    background-image: url(./Images/Header-desktop.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    padding: 168px 169px;
+    height: 815px;
+    background-size: cover;
+    width: 100%;
+  }
+
+  header {
+    padding: 28px 117px;
+  }
+
+  .mobile {
+    display: none;
+  }
+
+  .contact-wrapper-two {
+    background-image: url(/Images/contact-desktop.png);
+    background-repeat: no-repeat;
+    background-position-x: right;
+    padding-top: 50px;
+    background-size: cover;
+    width: 100%;
+  }
+
+  .project-card {
+    flex-direction: row;
+    gap: 20px;
+    margin-bottom: 84px;
+    border-radius: 18px;
+  }
+
+  .reverse {
+    flex-direction: row-reverse;
+  }
+
+  .project-img {
+    display: none;
+  }
+
+  .project-img img {
+    width: 100%;
+  }
+
+  .card-info {
+    width: 50%;
+    margin: 20px;
+  }
+
+  .project-btn {
+    margin: 15px 0;
+  }
+
+  .project-btn:hover {
+    border: 1px solid #396df2;
+    background: #396df2;
+    color: white;
+    width: 140px;
```

Comment:
- [ ] As the previous reviewer mentioned when you `hover` this button it increases its width. I think the problem could be because you apply a fixed width in the hover state of `140px` and normally it is `120px`. Please, kindly fix this issue for a better user experience :wink: :raised_hands:

| Normal state | Hover state |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/145679718-f7bfc506-4f57-431a-9e1c-cdda26bbeeab.png) | ![image](https://user-images.githubusercontent.com/81522435/145679741-7fb3a4cb-6159-4807-9e84-f7099b3c52df.png) |

Reply:
done